### PR TITLE
feat(About, Download): add Chinese translation

### PR DESCRIPTION
**Pull request by ChungZH**

### CODE REVIEW

File: content/zh/about/_index.html [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
 
 {{< /blocks/lead >}}
 
 {{< blocks/lead >}}
 
-This website is built by <a href="https://gohugo.io/" target="_blank">Hugo</a> with the <a href="https://www.docsy.dev/" target="_blank">Docsy</a> theme.
+该网站使用 <a href="https://gohugo.io/" target="_blank">Hugo</a> 以及 <a href="https://www.docsy.dev/" target="_blank">Docsy</a> 主题构建。
 
-The <a href="/">home page</a> is based on the <a href="https://colorlib.com/" target="_blank">Colorlib</a> template.
+这个 <a href="/">主页</a> 基于 <a href="https://colorlib.com/" target="_blank">Colorlib</a> 模板。
```
Comment (ouuan) [inline content/zh/about/_index.html:40]:
```suggestion
<a href="/">主页</a> 基于 <a href="https://colorlib.com/" target="_blank">Colorlib</a> 模板制作。
```

File: content/zh/download/_index.html [hunk 2]
```diff
@@ -19,11 +19,11 @@
     <section class="row td-box td-box--1 position-relative td-box--gradient td-box--height-auto">
         <div class="container text-center td-arrow-down">
             <span class="h4 mb-0">
-                <h1>Get CP Editor on your machine and have a try!</h1>
+                <h1>在你的计算机上获取 CP Editor，然后试一试！</h1>
                 <p v-if="latestStableAssetForUserPlatform" class="pt-3">
-                    You probably want to use
+                    你可能要使用
```
Comment (ouuan) [inline content/zh/download/_index.html:24]:
```suggestion
                    你可能想要使用
```

File: content/zh/download/_index.html [hunk 3]
```diff
@@ -33,43 +33,43 @@ <h1>Get CP Editor on your machine and have a try!</h1>
     <div class="container pt-5">
         <div class="row">
             <div class="col-lg-6 text-center py-3">
-                <h2>Choose your platform and the version you want to download</h2>
+                <h2>选择你要下载的平台上的版本</h2>
```
Comment (ouuan) [inline content/zh/download/_index.html:36]:
```suggestion
                <h2>选择你使用的平台和想要下载的版本</h2>
```

File: content/zh/download/_index.html [hunk 4]
```diff
@@ -33,43 +33,43 @@ <h1>Get CP Editor on your machine and have a try!</h1>
...
-                        <a href="https://github.com/cpeditor/cpeditor/releases" target="_blank">All versions</a>
+                        <a href="https://github.com/cpeditor/cpeditor/releases" target="_blank">所有版本</a>
                     </li>
                 </ul>
             </div>
             <div class="col-lg-6 py-3">
-                <h2 v-if="releases.length === 0">Loading...</h2>
-                <h2 v-else-if="!selectedAsset" class="text-warning">No available asset found</h2>
+                <h2 v-if="releases.length === 0">加载中...</h2>
+                <h2 v-else-if="!selectedAsset" class="text-warning">找不到可用文件</h2>
```
Comment (ouuan) [inline content/zh/download/_index.html:46]:
```suggestion
                <h2 v-else-if="!selectedAsset" class="text-warning">找不到可下载的文件</h2>
```

File: content/zh/download/_index.html [hunk 5]
```diff
@@ -33,43 +33,43 @@ <h1>Get CP Editor on your machine and have a try!</h1>
...
-                        <label for="asset">Version</label>
+                        <label for="asset">版本</label>
                         <select class="form-control" id="asset" v-model="selectedAsset">
                             <template v-for="release in releases" :key="release.tag_name">
                                 <template v-for="asset in release.assets" :key="asset.id">
                                     <option v-if="isAssetSuitableForPlatform(asset, selectedPlatform)" :value="asset">
                                         {{ asset.name }}
-                                        <span v-if="release === latestStableRelease">(Latest Stable)</span>
-                                        <span v-else-if="release === releases[0]">(Latest Beta)</span>
+                                        <span v-if="release === latestStableRelease">(最新稳定)</span>
```
Comment (ouuan) [inline content/zh/download/_index.html:61]:
```suggestion
                                        <span v-if="release === latestStableRelease">(最新稳定版)</span>
```

File: content/zh/download/_index.html [hunk 6]
```diff
@@ -33,43 +33,43 @@ <h1>Get CP Editor on your machine and have a try!</h1>
...
+                                        <span v-else-if="release === releases[0]">(最新预览)</span>
```
Comment (ouuan) [inline content/zh/download/_index.html:62]:
```suggestion
                                        <span v-else-if="release === releases[0]">(最新预览版)</span>
```

**Review by ouuan — CHANGES_REQUESTED** [review]

**Review by ouuan — APPROVED** [review]
LGTM

**Review by coder3101 — APPROVED** [review]

**Comment by ouuan** [comment]
@ChungZH Next time remember to squash and merge.

**Comment by ChungZH** [comment]
> @ChungZH Next time remember to squash and merge.

Ahhhhhhhhhhhhhh 💩

**Pull request merged**